lib/node_modules/@stdlib/math/base/special/cabsf/README.md

```diff
@@ -78,17 +78,13 @@ var y = cabsf( new Complex64( 5.0, 3.0 ) );
 
 ```javascript
 var Complex64 = require( '@stdlib/complex/float32/ctor' );
-var discreteUniform = require( '@stdlib/random/base/discrete-uniform' ).factory;
+var discreteUniform = require( '@stdlib/random/base/discrete-uniform' );
 var cabsf = require( '@stdlib/math/base/special/cabsf' );
 
-// Create a PRNG to generate uniformly distributed pseudorandom integers:
-var rand = discreteUniform( -50, 50 );
-
-// Compute the absolute value for a set of random numbers...
 var z;
 var i;
 for ( i = 0; i < 100; i++ ) {
-    z = new Complex64( rand(), rand() );
+    z = new Complex64( discreteUniform( -50, 50 ), discreteUniform( -50, 50 ) );
```

Comment:
What's the rationale for this change? I don't see any advantages compared to what was used previously.